app/validators/immigration_status_validator.rb

```diff
@@ -20,4 +16,25 @@ def course_can_sponsor_visa?(application_choice)
     (application_choice.course.salary? && application_choice.course.can_sponsor_skilled_worker_visa?) ||
       (!application_choice.course.salary? && application_choice.course.can_sponsor_student_visa?)
   end
+
+  def link_to_find
+    view.govuk_link_to('Find a course that has visa sponsorship', url_helpers.find_url, target: '_blank', rel: 'nofollow')
+  end
+
+private
+
+  def view
+    Class.new do
+      include ActionView::Helpers::UrlHelper
+      include GovukLinkHelper
+    end.new
+  end
+
+  def url_helpers
+    Rails.application.routes.url_helpers
+  end
```

Comment:
What are your thoughts on using `include Rails.application.routes.url_helpers` within these validators. The others don't appear to have the helper method?

Reply:
There are too many methods in `url_helpers` to include them all in a single class. This class is only using one of them.

Including the whole module would flood the object with methods it doesn't need.